src/components/share/modal/ShareModalExemple.tsx

```diff
@@ -0,0 +1,183 @@
+import { useEffect, useState } from "react";
+import { ShareModal } from "./ShareModal";
+
+import {
+  AccessData,
+  InvitationData,
+  UserData,
+} from ":/components/share/types.ts";
+import { ShareModalCopyLinkFooter } from "../utils/ShareModalCopyLinkFooter";
+import { DropdownMenuOption } from ":/components/dropdown-menu";
+
+type UserType = UserData<{
+  short_name?: string;
+  language?: string;
+}>;
+
+type InvitationType = InvitationData<
+  UserType,
+  {
+    surname: string;
+  }
+>;
+
+type AccessType = AccessData<
+  UserType,
+  {
+    name: string;
+  }
+>;
+
+export const ShareModalExemple = () => {
```

Comment:
exAmple*

+ WDYT about moving it in a dedicated folder as it concerns only storybook?